test/testHelpers.ts

```diff
@@ -136,3 +142,43 @@ export async function createCheck(applicant_id: string, document_id: string, ove
 
   return onfido.check.create(requestPayload);
 }
+
+export function sortByFirstName( a: Applicant, b: Applicant ) {
```

Comment:
I'd suggest that we make those method names more descriptive if possible 💄 

```suggestion
export function sortByApplicantFirstName( a: Applicant, b: Applicant ) {
```

And the same applied to the methods below.

Reply:
It'd done